packages/core/src/odata-common/entity-builder.spec.ts

```diff
@@ -206,9 +206,19 @@ describe('EntityBuilder', () => {
       const entity = TestEntity.builder().fromJson(entityJson);
 
       expect(entity).toStrictEqual(expectedEntity);
-      expect(entity.toSingleLink.getCustomFields()).toEqual(
-        expectedEntity.toSingleLink.getCustomFields()
+      expect(entity.toSingleLink!.getCustomFields()).toEqual(
```

Comment:
This could be a breaking change right? If someone accesses the `toSingleLink` it could be silently undefined in the past. Now we get a compile error. I think we should add a compatablity node in the changelog.md.